e2e/data/pipeline/live-channel-rules.json

```diff
@@ -0,0 +1,3 @@
+{
+  "rules": []
```

Comment:
`data/pipeline` should  not be included (yet) -- that requires a feature flag that we do not want to require

Reply:
done